recipes/mysql/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "mysql" %}
+{% set major_minor_version = "5.7" %}
+{% set micro_version = "20" %}
+{% set sha256 = "260582844ac18222ce2826f48b6c7ca387990b19ddb88331af26738b394e42a4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ major_minor_version }}.{{ micro_version }}
+
+source:
+  fn: mysql-boost-{{ major_minor_version }}.{{ micro_version }}.tar.gz
+  url: https://cdn.mysql.com/Downloads/MySQL-{{ major_minor_version }}/mysql-boost-{{ major_minor_version }}.{{ micro_version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - perl 5.22.2.1
+    - libedit
```

Comment:
So we typically use `readline` for this sort of thing. Admittedly we added `libedit` recently, but more as a crutch for a `conda` solver tar pit. Any thoughts on using `readline`?